### DISCUSSION TITLE
antsAI/antsRegistrationSyN.sh and ITK version incompatibility

### DISCUSSION THREAD

**Discussion by asylvester28** [ANTsX/ANTs#1803] [Q&A] [answered]
Hello everyone!

I am trying to use antsAI and antsRegistrationSyN.sh to register my T1-weighted structural image to MNI space. 

However, I am encountering a very persistent error in which it cannot find libITKIOBruker-5.3.1.dylib: 
_dyld[1909]: Library not loaded: @rpath/libITKIOBruker-5.3.1.dylib_ and a series of (no such file) errors for everywhere it has looked. There is a version of libITKIOBruker in one of the paths it tried (miniconda/ants_env/lib), but it is version 5.2.1.

I have tried installing and reinstalling ITK, including building it from source, but even with the latest version of ITK, the version of ITKIOBruker in my library remains 5.2.1. Is it possible to use version 5.2.1 with ANTs? If not (and I guess this would no longer be an ANTs question), can I update to 5.3.1 somehow?

I'm super new to ANTs and ITK, so apologies if I have phrased my question poorly. Please let me know if I should direct this question to ITK instead!

Thank you for your help :)

**Comment by cookpa** [comment]
Hi, can you please open a bug report issue and give details of your machine and how you installed ants?

**Reply by cookpa** [reply]
Meantime, a quick guess: if you installed ANTs with shared libraries, you might need to add `/path/to/ants/install/lib` to your `LD_LIBRARY_PATH`

**Reply by asylvester28** [reply, marked as answer]
Hi! Thank you for your quick reply! In beginning to fill out a bug issue, I discovered that I had version 5.2.0 of ANTs rather than 5.2.3 (even though I reinstalled from conda yesterday). Installing the latest MacOS 14 arm64 version seems to have solved the problem.
